bodhi/models/models.py

```diff
@@ -706,16 +707,6 @@ def new(cls, request, data):
                     build.package for build in data['builds']
                 ] if pkg.requirements], []))))
 
-        if not data['autokarma']:
-            del(data['stable_karma'])
-            del(data['unstable_karma'])
-            caveats.append({
-                'name': 'autokarma',
-                'description': 'Auto-karma stable requests disabled.',
-            })
-
-        del(data['autokarma'])
```

Comment:
@lmacken So I have removed all these from here like https://paste.fedoraproject.org/368261/46363292/ as you told earlier and the current patch is passing the current bodhi test suite along with the testcases which are included in this patch \o/.
So what should be our next step? :)